{"uuid": "be1c4509-f480-4de1-8b1d-81f8ce876f34", "children": ["c30a0625-028b-4230-bd26-c9cdc7142c8d"], "befores": [{"name": "get_lf_logs", "status": "passed", "start": 1702380868303, "stop": 1702380868303}], "afters": [{"name": "get_lf_logs::collect_logs_lf", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/allure_commons/_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/conftest.py\", line 941, in collect_logs_lf\n    obj.pull_file()\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/lanforge/scp_util.py\", line 22, in pull_file\n    ssh.connect(hostname=self.ip, username=self.username, password=self.password, port=self.port, allow_agent=False,\n  File \"/usr/local/lib/python3.8/dist-packages/paramiko/client.py\", line 435, in connect\n    self._auth(\n  File \"/usr/local/lib/python3.8/dist-packages/paramiko/client.py\", line 753, in _auth\n    self._transport.auth_password(username, password)\n  File \"/usr/local/lib/python3.8/dist-packages/paramiko/transport.py\", line 1564, in auth_password\n    return self.auth_handler.wait_for_response(my_event)\n  File \"/usr/local/lib/python3.8/dist-packages/paramiko/auth_handler.py\", line 240, in wait_for_response\n    event.wait(0.1)\n  File \"/usr/lib/python3.8/threading.py\", line 558, in wait\n    signaled = self._cond.wait(timeout)\n  File \"/usr/lib/python3.8/threading.py\", line 306, in wait\n    gotit = waiter.acquire(True, timeout)\n"}, "start": 1702380894642, "stop": 1702380895786}], "start": 1702380868303, "stop": 1702380895786}